SPECS/iputils/iputils.spec

```
@@ -32,7 +32,7 @@ The Iputils package contains programs for basic networking.
 %install
 %meson_install
 
-# rdisc and ninfod installed in sbin by default
+mkdir -p ${RPM_BUILD_ROOT}%{_sbindir}
```

I think we should use the macro, as it is used in the majority of the specs:
```suggestion
mkdir -p %{buildroot}/%{_sbindir}
```